### PR TITLE
issue new auth token if expired

### PR DESCRIPTION
**Pull request by maheshkurma**
Issue the user a new auth token if Expired.

### CODE REVIEW

File: src/Modules/Family/RegistrationContainer.js [hunk 1]
```diff
@@ -28,15 +28,17 @@ const RegistrationContainer = (props) => {
 
   const getUserToken = async () => {
     const localUserToken = localStorage.getItem('userToken');
-    if (!localUserToken || localUserToken === 'undefined') {
+    const tokenExpiresAt = localStorage.getItem('tokenExpiresAt');
+    if (new Date(tokenExpiresAt) < new Date() || !localUserToken || localUserToken === 'undefined') {
```
Comment (davidpickering) [inline src/Modules/Family/RegistrationContainer.js:32]: Not sure if I'm reading this correctly, but in my beta environment I only have 'userToken' in my local storage. I don't have a secondary 'tokenExpiresAt' value.
image

Comment (davidpickering) [reply src/Modules/Family/RegistrationContainer.js:32]: Is this code basically allows three different ways for someone to get a token set in their browser?

Comment (maheshkurma) [reply src/Modules/Family/RegistrationContainer.js:32]: Yes that's right david. I'm storing tokenExpiresAt value now in  local storage.
first time when a new user lands on registration form page, we will be issuing a new token with expiration to the user.
next if some one lands on same page later, we are checking 
if token is Expired or undefined ?
 issueNewToken
else
 useExistingUserToken

Comment (davidpickering) [reply src/Modules/Family/RegistrationContainer.js:32]: Thanks for the explanation @maheshkurma , that is exactly what we want!

**Pull request merged**